src/app/@ansyn/ansyn/app-effects/effects/cases.app.effects.ts

```diff
@@ -129,15 +134,16 @@ export class CasesAppEffects {
 			})
 		);
 
-	@Effect()
-	onLoadDefaultCase$: Observable<IBaseImageryLayer> = this.actions$.pipe(
+	@Effect({dispatch: false})
+	onLoadDefaultCase$ = this.actions$.pipe(
 		ofType(CasesActionTypes.LOAD_DEFAULT_CASE),
 		withLatestFrom(this.store$.select(selectMapsIds)),
 		filter(([action, [mapId]]: [LoadDefaultCaseAction, string[]]) => !action.payload.context && Boolean(mapId)),
-		mergeMap(([action, [mapId]]: [LoadDefaultCaseAction, string[]]) => {
```

Comment:
this effect was return an OL layer,
as return from `communicator.loadInitialMapSource(position)`
but in effect you should return an Action to be dispatch.
so no need to mergeMap in the end because i dont return an action,
and also dispatch: false was  require because i don't dispatch anything.